sql/helpers.py

```diff
@@ -12,3 +12,17 @@ def entity_to_dict(query_result):
         result = query_result.__dict__
         result.pop('_sa_instance_state', None)
     return result
+
+
+def helper_session(func):
+    def wrapper(self, *args, **kwargs):
+        self.temp_session = self.Session()
+        try:
+            return func(self, *args, **kwargs)
+        except:
+            self.temp_session.rollback()
+            raise
+        finally:
+            self.temp_session.close()
+    return wrapper
+

```

Comment:
Looks like than you can move _entity to dict(query)_ into this decorator

Reply:
Cannot agree, because we have methods for deleting, which don't return anything, and methods for task count, which return values that are not converted to dict

Reply:
ok